server/lib/groupBy.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
```

Comment:
If it isn't too much trouble, could we avoid camel case in file names? At least in our new files.